Resources/public/js/views/dashboard/ez-dashboardblockasynchronousview.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockasynchronousview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the All Content Dashboard Block View class
+     *
+     * @module ez-dashboardblockallcontentview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_ROW = '.ez-block-row',
+        SELECTOR_CONTENT = '.ez-block-content',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_ROW] = {tap: '_uiShowRowOptions'};
+
+    /**
+     * The asynchronous dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockAsynchronousView
+     * @constructor
+     * @extends eZ.DashboardBlockBaseView
+     */
+    Y.eZ.DashboardBlockAsynchronousView = Y.Base.create('dashboardBlockAsynchronousView', Y.eZ.DashboardBlockBaseView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._getContent;
+            this._watchAttribute = 'content';
+            /**
+             * Stores the click outside event handler
+             *
+             * @property _clickOutsideHandler
+             * @type {Object}
+             * @protected
+             */
+            this._clickOutsideHandler = null;
+            this.events = Y.merge(this.events, EVENTS);
+
+            this.get('container')
+                .addClass(this._generateViewClassName(Y.eZ.DashboardBlockBaseView.NAME))
+                .addClass(CLASS_LOADING);
+
+            this.after('contentChange', this._uiEndLoading);
+        },
+
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockAllContentView} the view itself
+         */
+        render: function () {
```

Comment:
you can completely remove that method, `Y.View` already provides exactly the same implementation.